scripts/validate_docstrings.py

```diff
@@ -311,6 +321,10 @@ def validate_one(func_name):
         for param_err in param_errs:
             errs.append('\t{}'.format(param_err))
 
+    mentioned_errs = doc.mentioned_private_classes
+    if mentioned_errs:
+        errs.append('Private classes mentioned: {}'.format(mentioned_errs))
```

Comment:
May be for junior people it'd be more useful a message like "Private classes {} should not be mentioned in public API docstrings". May be I'm underestimating people capacity to understand error messages, but I anticipate people asking what they need to do about this error message.

Reply:
Good suggestion! I was thinking on a longer message. Because in principle I can even say what they should replace it with, but that will maybe become a bit too complicated